### ISSUE TITLE
Bootstrap RTL (ping)

### ISSUE THREAD

**Issue by powerchelle** [mdo/ama#290]
Hi Mark, any update on the plan to integrate RTL changes for next minor release? We have a big product release gating on this plan, and several months have passed since the last update/reply from you. DO we have a target release with date for the RTL changes, or do you recommend using something like [this](http://bootstrap.rtlcss.com/) instead?

[Original AMA issue](https://github.com/mdo/ama/issues/285)

**Comment by powerchelle** [comment]
ping again?

**Comment by mdo** [comment]
Sorry @powerchelle, I've been heads down on GitHub work the last few weeks. For now, I'd suggest going with a third party solution to it. We've been focused on adding the components we've always wanted in Bootstrap while also clearing out the cruft.

We've never been able to internally tackle RTL and I don't believe anyone on the team has had first hand experience (I've designed for it at Twitter, but didn't build the CSS system for it). That's made it difficult for us to really know what the best solution would be for the most people, so we've constantly punted on the PRs we've received.

I've raised it with the team again to see what might be doable after v4.3 ships this coming Monday.
